### ISSUE TITLE
getNearbyEntities not working in 7.4

### ISSUE THREAD

**Issue by Russia9** [SpongePowered/SpongeAPI#1929]
I do not know where to write about it, so I will write here. When I switched from **API v5** to **API v7** my plugin stopped working correctly. It turned out the problem was with the method `Entity.getNearbyEntities(double distance)`. Moreover, the problem was caused only with SpongeForge, SpongeVanilla is working correctly. Sorry for my english, i'm from Russia.

**Comment by XakepSDK** [comment]
Dupe: https://github.com/SpongePowered/SpongeCommon/issues/1607

**Comment by ImMorpheus** [comment]
@Russia9 Can you provide more information ? You haven't explained why `Entity.getNearbyEntities(double distance)`is not working

**Comment by Russia9** [comment]
I am writing a simple plugin for chat management. In it i have `MessageChannelEvent.Chat` Listener. It works exactly, I checked it. Inside it i have this code:
`event.setChannel(MessageChannel.fixed(player.getNearbyEntities(plugin.getChatSpongeConfig().chatRange).stream().filter(Player.class::isInstance).map(Player.class::cast)` Experimentally, I determined that the problem with this fragment of the code. In API v5 or SpongeVanilla API v7 it works correctly, but in SpongeForge it doesn't works.
